src/pybind/mgr/volumes/module.py

```diff
@@ -49,8 +49,10 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
         {
             'cmd': 'fs volume create '
                    f'name=name,type=CephString,goodchars={goodchars} '
-                   'name=placement,type=CephString,req=false ',
-            'desc': "Create a CephFS volume",
+                   'name=placement,type=CephString,req=false '
+                  f'name=meta_pool,type=CephString,goodchars={goodchars},req=false '
+                  f'name=data_pool,type=CephString,goodchars={goodchars},req=false ',
```

Comment:
FWIW, there is `type=CephPoolname` used in mgr/mon commands. I wan't aware of it till now.

Reply:
Not in `fs new` command - https://github.com/ceph/ceph/blob/main/src/mon/MonCommands.h#L358-L359

Reply:
Weird, it's same as string - https://github.com/ceph/ceph/blob/main/src/mon/MonCommands.h#L65

Reply:
CephFS commands are using CephString in most places, let's keep CephSrtring here too?